docs/disks/spec2006/runscript.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
```

Comment:
Could you add a comment here describing this file?

For instance:
  This file is the script that runs on the gem5 guest. This reads a file from the host via m5 readfile
  which contains the workload and the size to run. Then it resets the stats before running the workload.
  Finally, it exits the simulation after running the workload, then it copies out the result file to be checked.